server/app/api.py

```diff
@@ -790,6 +791,8 @@ class AssignmentAPI(APIResource):
             'methods': set(['POST']),
             'web_args': {
                 'grade_final': BooleanArg(),
+                'testing': BooleanArg(),
```

Comment:
Indeed. At some point I'll add this to this flag to the frontend (the flag tells the autograder to not send grades back)

Actually, we should just have a hide grades feature.